tools/nodejs_api/sample.js

```diff
@@ -0,0 +1,26 @@
+// Make sure the test directory is removed as it will be recreated
+const fs = require("fs");
+const {Database, Connection} = require("./build/kuzu");
+
+try {
+  fs.rmSync("./test", { recursive: true });
+} catch (e) {
+  // ignore
```

Comment:
Actually I wrote this. Since this test script always recreate a test database, we remove the old one before running, but node throws an error when removing non-existent file, which can be ignored.